WireAuthentication/Sources/WireAuthentication/Components/DetermineAuthMethodComponent.swift

```diff
@@ -55,12 +56,13 @@ class DetermineAuthMethodComponent: Component<DetermineAuthMethodComponentDepend
         email: String?,
         canCreateAccount: Bool,
         didDetectDomainConflict: Bool,
-        backendInfo: BackendInfo
```

Comment:
Here and elsewhere you'll see that `BackendInfo` has been replaced by `BackendEnvironment2` (which will be renamed to `BackendEnvironment`), which will be the sole environment model going forward.